SourceCpp/React.cpp

```diff
@@ -103,6 +128,7 @@ PeleC::react_state(
       const int do_update = react_init ? 0 : 1;
 
       const int captured_clean_react_massfrac = clean_react_massfrac;
+      const int captured_chem_integrator = chem_integrator;
```

Comment:
Can you reduce the scope of this so it doesn't cause an unused variable warning?

Reply:
I moved it into the sundials section